docs/build_release.md

```diff
@@ -1,5 +1,59 @@
 # Build and release process
 
+Presidio leverages Azure DevOps YAML pipelines to validate, build, releasee and deliver presidio. the pipelines make use of [templates](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops) for code reuse using [YAML Schema](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema).
+
 ## Description
 
-## Table of contents
+The following pipelines are provided and maintained as part of presidio development process:
+
+* [PR Validation](../azure-pipelines.yml) - used to validate pull requests.
+    * Linting
+    * Credential Scanning
```

Comment:
we also do Component Governance detection and code analysis. This item should be just names security and compliance analysis

Reply:
done